src/Tasks/GenerateResource.cs

```diff
@@ -3054,8 +3052,26 @@ private void ReadResources(String filename, bool shouldUseSourcePath, String out
                             foreach (XElement dataElem in doc.Element("root").Elements("data"))
                             {
                                 string name = dataElem.Attribute("name").Value;
+                                string typeName = dataElem.Attribute("type")?.Value;
                                 string value = dataElem.Element("value").Value;
-                                AddResource(reader, name, value, filename);
+
+                                if (typeName != null)
+                                {
+                                    int pos = typeName.IndexOf(",", StringComparison.Ordinal);
```

Comment:
Minor comment - I still wonder why you use "," _string_ if you could use ',' char.
Below too.

Reply:
Because it doesn't build if I use the char. 😉

I found this out after I applied your two suggested fixes from earlier. I had to commit 8f448bc to fix the problem we inadvertently introduced.

Reply:
Sorry, there is an overload for char `IndexOf(char ch)`.

Reply:
But not if you specify the StringComparison value as well, which is what you suggested above. I admit I couldn't quite decide between keeping the char or using the StringComparison; I chose the StringComparison. I guess I could change it back, especially since the char being tested isn't an alphabetic character, so locale/case-sensitivity would not apply.

Reply:
Sorry again, I think it is ok to ignore a locale/case-sensitivity and use `IndexOf(char ch)`.